content/influxdb/clustered/install/configure-cluster.md

```diff
@@ -733,12 +772,12 @@ in your `myinfluxdb.yml`. If omitted, your cluster will use the default scale se
 - [CPU resource units](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu)
 - [Memory resource units](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory)
 
-
 {{% code-placeholders "(INGESTER|COMPACTOR|QUERIER|ROUTER)_(CPU|MEMORY|REPLICAS)" %}}
+
 ```yml
 apiVersion: kubecfg.dev/v1alpha1
 kind: AppInstance
-...
+---
```

Comment:
Did you intend to replace ellipses with dashes?

Reply:
Nope, auto-formatter. I think it did this because the ellipsis isn't valid yaml (it needs to be commented out).